tests/operators/arithmetic_operators.cpp

```diff
@@ -35,11 +35,11 @@ TEST_CASE("Arithmetic operators") {
         }
     }
     {  //  +
-        auto rows = storage.select(columns(c(&Object::nameLen) + 1000));
+        auto rows = storage.select(c(&Object::nameLen) + 1000);
```

Comment:
after this change current test looks the same like the previous one. This test with a `column` with a single argument was created intentionally to cover corner case. Better bring it back

Reply:
Sorry about that!